README.md

```diff
@@ -20,7 +20,7 @@ Usage of vpc-network-config-analyzer:
   -debug
         Run in debug mode
   -dst string
-        Destination name for network_interface or an external IP to be explained
+        Destination name for explaination; can be specified as a vsi name or an internal or external address (IP/cidr)
```

Comment:
update based on additional changes in `parse_args.go`

Reply:
done. I've also changed dst to no reference src